internal/matrix/matrix_test.go

```diff
@@ -59,69 +59,53 @@ func TestConvertMaptoInts(t *testing.T) {
 	}
 }
 
-func TestClientErrorHandling(t *testing.T) {
+func TestGenerateFibMatrix(t *testing.T) {
 
 	data := []struct {
 		name     string
 		rows     int
 		columns  int
-		expected error
+		expected [][]int64
+		err		 error
```

Comment:
are you running a go formatter with your IDE? this doesn't seem to be following formatting